src/server/index.jsx

```diff
@@ -206,9 +206,13 @@ server.get(
       } = getRouteProps(urlPath);
 
       const { page, renderer_env } = query;
-
+      const isOptimo = urlPath.match(/(c[a-zA-Z0-9]{10,}o)/) > 0;
+      const cafEnabledService =
+        ['thai', 'pidgin', 'hausa'].includes(service) && !isOptimo;
       const isCaf = !!(
-        renderer_env === 'caftest' || renderer_env === 'caflive'
+        cafEnabledService ||
+        renderer_env === 'caftest' ||
+        renderer_env === 'caflive'
```

Comment:
I'd maybe lift the `cafEnabledService` flag outside of the `!!(` as its a boolean anyway.

Something like this may be a bit clearer:
```
      const isCaf =
        cafEnabledService ||
        Boolean(renderer_env === 'caftest' || renderer_env === 'caflive');
```